lgc/interface/lgc/LgcContext.h

```diff
@@ -129,6 +138,7 @@ class LgcContext {
   llvm::TargetMachine *m_targetMachine = nullptr; // Target machine
   TargetInfo *m_targetInfo = nullptr;             // Target info
   unsigned m_palAbiVersion = 0xFFFFFFFF;          // PAL pipeline ABI version to compile for
+  bool m_robustAccessEx = false;                  // robustAccessEx option from last pipeline compile
```

Comment:
Please do not add a pipeline option into LgcContext. It belongs in Options in Pipeline.h. In Options, it can just be called robustAccess rather than robustAccessEx, as the old robust access option is not in LGC at all.

Reply:
Thanks. I will fix it.